packages/v4/src/content/design-guidelines/components/code-block/code-block.md

```diff
@@ -0,0 +1,34 @@
+---
+id: Code block
+section: components
+---
+
+The **code block** is a component that contains two or more lines of ready-only code. The code in a code block can be copied to the clipboard.
+
+## Elements
+Code block has four main components.
+
+<img src="./img/code-block-structure.png" alt="Elements of the code block component" width="441"/>
+
+1. **Code snippet:**  the content that will be copied to the user’s clipboard.
+2. **Clipboard button:** completes the copy action when clicked. The copy button is always present but additional actions can be added on an as needed basis.
+3. **Tooltip:** informs users that clicking the button will copy the content to their clipboard, or that the content has successfully been copied.
+4. **Show more/less (optional):** expands and collapses the code snippet.
+
+**Show more/less**
```

Comment:
Can you make the H3s? So just pre-face with 3 hashtags

Reply:
Yes I can make those H3s.